### DISCUSSION TITLE
the meaning of minting

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#439] [Q&A] [answered]
Hello,

Yesterday when I learning lesson 11 about nft, I deployed nft on testnet opensea, and on event it's written **Minted** ,

Tbh, I don't really understand, what the meaning of minted

Could someone help to explain to me?

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @gunomnifluencer I'll suggest you to read [this](https://azbigmedia.com/business/how-nft-minting-works-an-initial-guide-to-nfts/) article it has detailed explanation about minting.

If you want a simpler answer, to mint a NFT means uniquely publishing your token (whit  it's URI and metadata) on a blockchain and also make in purchaseble.

Best regards, Cromewar.

**Reply by gunomnifluencer** [reply]
Hello,

Thanks for reply,

Just ask is it possible to spreading funds into more than 1 wallet after user buy **nft** from us?

**Reply by cromewar** [reply]
Yes but not on a single transaction, each one are made from an address to another address, you must execute a withdraw function independently for each wallet.
